apollo-normalized-cache-sqlite/src/androidMain/kotlin/com/apollographql/apollo/cache/normalized/sql/internal/IOTaskExecutor.kt

```diff
@@ -0,0 +1,16 @@
+package com.apollographql.apollo.cache.normalized.sql.internal
+
+import com.apollographql.apollo.cache.normalized.sql.ApolloDatabase
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+/**
+ * Performs IO operation on background thread via [Dispatchers.IO].
+ */
+internal actual class IOTaskExecutor {
+  actual suspend fun <R> execute(operation: () -> R): R {
+    return withContext(Dispatchers.IO) {
+      operation()
+    }
```

Comment:
Not for myself: duplicating this in androidMain and jvmMain feels bad. I think this is something solved by HMPP once I get a chance to look into it.

Reply:
yeah, it's annoying